### PR TITLE
DBZ-4461 Add documentation for postgres bytea_output

### PR DESCRIPTION
**Pull request by hjwalt**
@gunnarmorling @jpechane adding docs for `bytea_output`. Should I also add something on the `binary.handling.mode` configuration table row?

https://issues.redhat.com/browse/DBZ-4461

### CODE REVIEW

File: documentation/modules/ROOT/pages/connectors/postgresql.adoc [hunk 1]
```diff
@@ -1421,7 +1421,9 @@ The string representation of the interval value that follows the pattern `P<year
 |`BYTES` or `STRING`
 |n/a +
  +
-Either the raw bytes (the default), a base64-encoded string, or a hex-encoded string, based on the connector's {link-prefix}:{link-postgresql-connector}#postgresql-property-binary-handling-mode[binary handling mode] setting.
+Either the raw bytes (the default), a base64-encoded string, or a hex-encoded string, based on the connector's {link-prefix}:{link-postgresql-connector}#postgresql-property-binary-handling-mode[binary handling mode] setting. +
```
Comment (gunnarmorling) [inline documentation/modules/ROOT/pages/connectors/postgresql.adoc:1424]: Could you use this link style, we're in the process of phasing out the other one:

```suggestion
Either the raw bytes (the default), a base64-encoded string, or a hex-encoded string, based on the connector's xref:postgresql-property-binary-handling-mode[binary handling mode] setting. +
```

File: documentation/modules/ROOT/pages/connectors/postgresql.adoc [hunk 2]
```diff
@@ -1421,7 +1421,9 @@ The string representation of the interval value that follows the pattern `P<year
...
+ +
+Debezium only supports Postgres `bytea_output` configuration of value `hex`.
```
Comment (gunnarmorling) [inline documentation/modules/ROOT/pages/connectors/postgresql.adoc:1426]: Could you add a link to the PG docs discussing this option?

**Review by gunnarmorling — COMMENTED** [review]
LGTM, two small suggestions inline.

**Comment by gunnarmorling** [comment]
> Should I also add something on the binary.handling.mode configuration table row?

I don't think that's needed.

File: documentation/modules/ROOT/pages/connectors/postgresql.adoc (hunk 1 above)
Comment (hjwalt) [reply documentation/modules/ROOT/pages/connectors/postgresql.adoc]: Updated!

File: documentation/modules/ROOT/pages/connectors/postgresql.adoc (hunk 2 above)
Comment (hjwalt) [reply documentation/modules/ROOT/pages/connectors/postgresql.adoc]: Updated!

**Review by gunnarmorling — APPROVED** [review]
LGTM, merging. Thx, @hjwalt!

**Pull request merged**